source/stylesheets/base/_typography.scss

```diff
@@ -73,3 +90,4 @@ time {
   letter-spacing: .75px;
```

Comment:
I usually try and use ems for letter-spacing. That way it's relative to the font-size and less brittle. You can imagine if the font-size was 72. You wouldn't notice the .75px spacing at all.